aten/src/ATen/native/native_functions.yaml

```diff
@@ -3657,15 +3657,25 @@
 
 - func: adaptive_avg_pool3d(Tensor self, int[3] output_size, *, Tensor(a!) out) -> Tensor(a!)
   python_module: nn
+  dispatch:
+    CPU: adaptive_avg_pool3d_out_cpu
+    CUDA: adaptive_avg_pool3d_out_cuda
 
 - func: adaptive_avg_pool3d(Tensor self, int[3] output_size) -> Tensor
   python_module: nn
+  dispatch:
+    CPU: adaptive_avg_pool3d_cpu
+    CUDA: adaptive_avg_pool3d_cuda
 
 - func: adaptive_avg_pool3d_backward(Tensor grad_output, Tensor self, *, Tensor(a!) grad_input) -> Tensor(a!)
-  python_module: nn
```

Comment:
I'm curious, why did you remove `python_module: nn` here but not above? Apparently it builds just fine, but most other locations in this file still have `python_module: nn`.

Reply:
Oh, this is my fault. I did not remove this deliberately and I get this back for consistency with others.